src/main/java/edu/greenblitz/robotName/subsystems/shooter/FlyWheel/FlyWheelConstants.java

```diff
@@ -8,15 +8,15 @@ public class FlyWheelConstants {
 	
 	public static final double EPSILON_RPM = 50;
 
-	public static final double DEFAULT_VELOCITY = 50;
+	public static final double DEFAULT_VELOCITY = 30;
 
 	public static final double COLLECT_FORM_FEEDER_POWER = -0.1;
 	
 	public static final double MINIMUM_SHOOTING_SPEED_TIME_ROBORIO_CYCLES = 7;
 	
 	public static final double SHOOTING_POWER = 0.9;
 	
-	public static final double SHOOTING_VELOCITY = 100;
+	public static final double SHOOTING_VELOCITY = 80;// (rps) new transmission (36/40):77.13 , old transmission (42/36): 100;
```

Comment:
??

Reply:
Adjusted flywheel speed after the transmission change, the speed we set before d2 (100) is equivalent to the new speed (77 or practically 80) in the new transmission.